sys/sys/systm.h

```diff
@@ -594,6 +594,7 @@ int	suword_c(volatile void * __capability base, long word);
 int	suword16_c(volatile void * __capability base, int word);
 int	suword32_c(volatile void * __capability base, int32_t word);
 int	suword64_c(volatile void * __capability base, int64_t word);
+int	sucap(volatile const void * __capability base, intcap_t val);
```

Comment:
I do sometimes wonder if this wants to be `suptr()`, but it's not a perfect fit.

Reply:
What I thought about was having a `suptr` wrapper that took a `void *`.  It would just use a cast to call `sucap` which would make some bits more readable.  We already have `fuecap` so `sucap` is probably more consistent for now.   I would not be opposed though to having some kind of `fueptr` and `suptr`.  It would make upstream FreeBSD more readable I think even.